app/controllers/articles_controller.rb

```diff
@@ -54,11 +54,7 @@ def new
 
     @article, needs_authorization = Articles::Builder.new(@user, @tag, @prefill).build
 
-    if needs_authorization
-      authorize Article
-    else
-      skip_authorization
-    end
+    needs_authorization ? authorize(Article) : skip_authorization
```

Comment:
Looks good to me. One question I have as I'm still a Ruby newb, but are parentheses required when it's ternary?

```suggestion
    needs_authorization ? authorize Article : skip_authorization
```

Reply:
I tried to use it without parentheses, but I got an error on tests. So I resolve this with the parentheses. :)